src/openapi/openapi.yml

```diff
@@ -351,16 +351,95 @@ components:
         started_at:
           type: string
           format: date-time
-          x-nullabe: true
+          x-nullable: true
         finished:
           type: boolean
         finished_at:
           type: string
           format: date-time
-          x-nullabe: true
+          x-nullable: true
         failed:
           type: boolean
 
+    TowerWebhookNotification:
```

Comment:
Is notification payload the same also for workflow jobs? Workflows don't have fields like project, playbook, and maybe few others.

Reply:
Added the following for job-workflows:
https://github.com/resource-hub-dev/rhub-api/pull/26/files#diff-3d6fdebe82daa0e03f81cf6f62f17d0c33268587a8914d89d7b4dffef2d83b99R446-R449